packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -401,7 +405,54 @@ export type InsertableTreeFieldFromImplicitField<
 	? ApplyKind<InsertableTreeNodeFromImplicitAllowedTypes<Types>, Kind, true>
 	: TSchema extends ImplicitAllowedTypes
 		? InsertableTreeNodeFromImplicitAllowedTypes<TSchema>
-		: unknown;
+		: never;
+
+/**
+ * {@inheritdoc (UnsafeUnknownSchema:type)}
+ * @alpha
+ */
+export const UnsafeUnknownSchema: unique symbol = Symbol("UnsafeUnknownSchema");
+
+/**
+ * A special value which some APIs accept as an alternative to a schema to indicate the schema can not easily be provided at compile time.
+ * @remarks
+ * When used this means the TypeScript typing should error on the side of completeness (allow all inputs that could be valid).
+ * This introduces the risk that out of schema data could be allowed at compile time, and only error at runtime.
+ *
+ * This only applies to APIs which input data which is expected to be in schema, since APIs outputting have easy mechanisms to do so in a type safe way even when the schema is unknown.
```

Comment:
I don't think this sentence and the one just below will be particularly interesting to a reader - the key point is that it applies to input APIs. I would just remove these (or move them to the private remarks if you think it's useful for devs), and note in the summary section of the doc that this applies to input APIs (as suggested above).

Reply:
made privateRemarks.